tests/integration/hive_metastore/test_migrate.py

```diff
@@ -101,3 +102,46 @@ def test_migrate_external_table(ws, sql_backend, inventory_schema, make_catalog,
 
     target_tables = list(sql_backend.fetch(f"SHOW TABLES IN {dst_schema.full_name}"))
     assert len(target_tables) == 1
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_revert_migrated_table(ws, sql_backend, inventory_schema, make_schema, make_table):
+    schema1 = make_schema(catalog_name="hive_metastore")
+    schema2 = make_schema(catalog_name="hive_metastore")
+    tables = [
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema2.name),
+        make_table(schema_name=schema2.name),
+    ]
+
+    for i in range(3):
+        logger.info(f"Applying upgraded_to table property for {tables[i].full_name}")
+        sql_backend.execute(f"ALTER TABLE {tables[i].full_name} set TBLPROPERTIES('upgraded_to'='FAKE DEST')")
+
+    crawler = TablesCrawler(sql_backend, inventory_schema)
+    tm = TablesMigrate(crawler, ws, sql_backend)
+    tm.revert_migrated_tables(schema=schema1.name)
+
+    def checks_upgraded_to(table_name: str):
```

Comment:
Add this as a method to TablesMigrate class

Reply:
OK